modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -335,10 +350,18 @@ const Main = ({
 	}, [allowGuestUsers, isSignedIn, showUploadPermissionMessage]);
 
 	useEffect(() => {
-		setCurrentValue(value);
-		setDisplayErrors(initialDisplayErrors);
-		setErrorMessage(getErrorMessages(initialErrorMessage, isSignedIn));
-		setValid(initialValid);
+		const objectInvalidExtension = isObjectInvalidExtension(value);
```

Comment:
@nicolas-sss Could you please rename `objectInvalidExtension` to `objectFieldInvalidExtension`.

Reply:
Done